.github/workflows/cleanup-snapshots.yml

```diff
@@ -0,0 +1,45 @@
+name: Cleanup Snapshot Artifacts on a CRON Schedule
+
+on:  
+  schedule:
+    # Runs "At 00:00 on day-of-month 1 in every 3rd month." (see https://crontab.guru)
+    - cron: '0 0 1 */3 *'
```

Comment:
This is cplace's release cycle.

![Screenshot 2023-04-28 at 08 36 20](https://user-images.githubusercontent.com/16098085/235072791-c2549f5b-68b9-4511-a89c-3ddeb319e83c.png)

This cron value would be to close to the release. It would be better if we set it to `0 0 28 */3 *`.

The job would then run on:

2023-07-28 
2023-10-28 
2024-01-28
2024-04-28 

And that should fit better IMO